keyboards/preonic/keymaps/default/keymap.c

```diff
@@ -260,40 +260,43 @@ void encoder_update_user(uint8_t index, bool clockwise) {
 }
 
 void dip_switch_update_user(uint8_t index, bool active) {
-  switch (index) {
-    case 0:
-      if (active) {
-        layer_on(_ADJUST);
-      } else {
-        layer_off(_ADJUST);
-      }
-      break;
-    case 1:
-      if (active) {
-        muse_mode = true;
-      } else {
-        muse_mode = false;
-        #ifdef AUDIO_ENABLE
-          stop_all_notes();
-        #endif
-      }
-   }
+    switch (index) {
+        case 0:
+            if (active) {
+                layer_on(_ADJUST);
+            } else {
+                layer_off(_ADJUST);
+            }
+            break;
+        case 1:
+            if (active) {
+                muse_mode = true;
+            } else {
+                muse_mode = false;
+            }
+    xs}
```

Comment:
@drashna seems you missed this 😉

Reply:
Oh no :(

And why did Travis not catch that, either?

Reply:
```
Making all keymaps for preonic
QMK Firmware 0.7.13
make: *** No rule to make target 'preonic:all'. Stop.
|
|  QMK's make format recently changed to use folder locations and colons:
|     make project_folder:keymap[:target]
|  Examples:
|     make planck/rev4:default:dfu
|     make planck:default
|
The command "bash util/travis_build.sh" exited with 0.
```

😕

Reply:
that ... would be why.  Opened another PR to fix that, and a couple of other issues that snuck by